meinberlin/apps/plans/views.py

```diff
@@ -50,6 +52,10 @@ class PlanDetailView(rules_mixins.PermissionRequiredMixin, CanonicalURLDetailVie
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["polygon"] = settings.BERLIN_POLYGON
+        serializer = project_serializers.ProjectSerializer(
+            self.object.published_projects, many=True, now=timezone.now()
+        )
+        context["published_projects"] = json.dumps(serializer.data)
```

Comment:
@sevfurneaux did you try to add the `published_projects` to the [Plan serializer](https://github.com/liqd/a4-meinberlin/blob/main/meinberlin/apps/plans/serializers.py#L10) instead? I'd advice to do so, as this is rather hacky here. 
Then please a line to assert the list of the published projects [here](https://github.com/liqd/a4-meinberlin/blob/main/tests/plans/test_serializer.py#L88).
Let me know if it's too much and I will take over this in another PR.

Reply:
I will see if I have time later or tomorrow morning. Otherwise I will merge, and make a new issue for later.

Reply:
Excellent, thanks!